MAUI/Scheduler/migration.md

```diff
@@ -199,6 +263,66 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
 ## Properties
 
 ### SfSchedule
+
+The following code example, explains how to initialize the schedule in Xamarin SfSchedule and .NET MAUI SfScheduler.
```

Comment:
The following code example, explains how to initialize the properties of the schedule in